scenarios/azure/fakeintake/fakeintake.go

```diff
@@ -0,0 +1,84 @@
+package fakeintake
+
+import (
+	"github.com/DataDog/test-infra-definitions/common/config"
+	"github.com/DataDog/test-infra-definitions/common/utils"
+	"github.com/DataDog/test-infra-definitions/components"
+	"github.com/DataDog/test-infra-definitions/components/command"
+	"github.com/DataDog/test-infra-definitions/components/datadog/fakeintake"
+	"github.com/DataDog/test-infra-definitions/components/docker"
+	"github.com/DataDog/test-infra-definitions/components/os"
+	"github.com/DataDog/test-infra-definitions/resources/azure"
+	"github.com/DataDog/test-infra-definitions/scenarios/azure/compute"
+	app "github.com/pulumi/pulumi-azure-native-sdk/app/v2"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func NewContainerAppInstance(e *azure.Environment) (*fakeintake.Fakeintake, error) {
+	return components.NewComponent(e, "fakeintake", func(fi *fakeintake.Fakeintake) error {
+		// environment, err := app.NewManagedEnvironment(e.Ctx(), "fakeintake-2", &app.ManagedEnvironmentArgs{
+		// 	ResourceGroupName: pulumi.String(e.DefaultResourceGroup()),
+		// 	VnetConfiguration: &app.VnetConfigurationArgs{
+		// 		Internal:               pulumi.Bool(true),
+		// 		InfrastructureSubnetId: pulumi.String(e.DefaultSubnet()),
+		// 	},
+		// }, e.WithProviders(config.ProviderAzure))
+
+		// if err != nil {
+		// 	return err
+		// }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Is this commented section a leftover?
```suggestion
```

Reply:
Oh yeah it was supposed to be deleted. Removing it now